deployment/CP05MOAS-GL340_Deploy.csv

```diff
@@ -59,3 +59,9 @@ TS20191212,,AR42-02,,CP05MOAS-GL340-03-CTDGVM000,10,1,2019-12-13T01:04:47,2020-0
 TS20191212,,AR42-02,,CP05MOAS-GL340-04-DOSTAM000,10,1,2019-12-13T01:04:47,2020-03-03T12:00:00,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00340,CGINS-DOSTAM-00398,40.0817,-70.74813,,200,200,EB 
 TS20191212,,AR42-02,,CP05MOAS-GL340-05-PARADM000,10,1,2019-12-13T01:04:47,2020-03-03T12:00:00,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00340,CGINS-PARADM-50145,40.0817,-70.74813,,200,200,EB 
 TS20191212,,AR42-02,,CP05MOAS-GL340-01-ADCPAM000,10,1,2019-12-13T01:04:47,2020-03-03T12:00:00,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00340,CGINS-ADCPAM-48878,40.0817,-70.74813,,200,200,EB 
+TS20200925,,,,CP05MOAS-GL340-00-ENG000000,11,1,2020-09-26T06:14:00,,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00340,CGCON-GLDRCP-00340,40.0667,-70.18002,,200,200,EB 
+TS20200925,,,,CP05MOAS-GL340-02-FLORTM000,11,1,2020-09-26T06:14:00,,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00340,CGINS-FLORTM-04467,40.0667,-70.18002,,200,200,EB 
+TS20200925,,,,CP05MOAS-GL340-03-CTDGVM000,11,1,2020-09-26T06:14:00,,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00340,CGINS-CTDGVM-09363,40.0667,-70.18002,,200,200,EB 
+TS20200925,,,,CP05MOAS-GL340-04-DOSTAM000,11,1,2020-09-26T06:14:00,,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00340,CGINS-DOSTAM-00361,40.0667,-70.18002,,200,200,EB 
+TS20200925,,,,CP05MOAS-GL340-05-PARADM000,11,1,2020-09-26T06:14:00,,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00340,CGINS-PARADM-50238,40.0667,-70.18002,,200,200,EB 
+TS20200925,,,,CP05MOAS-GL340-01-ADCPAM000,11,1,2020-09-26T06:14:00,,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00340,CGINS-ADCPAM-72777,40.0667,-70.18002,,200,200,EB 
```

Comment:
Looks good.  Checked that latest Cal csvs already exist for all sensors.

Reply:
CUID, Lat/Lon, and S/Ns look good.  Deployment time is not consistent in other documentation. That needs to be finalized before this time can be checked.

Reply:
Deployment time confirmed.